src/components/Map.js

```diff
@@ -407,25 +402,55 @@ class MapComponent extends Component {
         latlng
       )
     );
-    
-    const wmsUrlsPromises = wmsUrls.map(url => getFromUrl(url)); 
-    Promise.all(wmsUrlsPromises).then(promiseResults => {
+    const urlsPromises = rasterUrls ? rasterUrls.map(url => getFromUrl(url)) : wmsUrls.map(url => getFromUrl(url));
+    Promise.all(urlsPromises).then(promiseResults => {
       if (
         this.state.featureInfo.latlng.lat === latlng.lat &&
         this.state.featureInfo.latlng.lng === latlng.lng
-        ) {
-          this.setState({
-            featureInfo: {
-              show: true,
-              latlng: latlng,
-              data: promiseResults,
-            }
-          });
+      ) {
+        this.setState({
+          featureInfo: {
+            show: true,
+            latlng: latlng,
+            data: promiseResults,
+          }
+        });
       }
     });
   }
 
-  
+  renderRasterInfoOrWmsInfo() {
+    //Check if there is raster in the tile
+    const rasters = this.props.tile.rasters && this.props.tile.rasters.map(raster => this.props.rasters.data[`${raster.uuid}`]);
+
+    return rasters ?
```

Comment:
What if rasters is a empty array ? 
Personally I would try to be more explicit like:  if(rasters && rasters.length > 0)

Reply:
This line is no longer exist